IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcGeographicCRS/DocEntity.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcGeographicCRS" UniqueId="2d6be663-8d5e-42cc-b5b4-13bb37f41262" BaseDefinition="IfcCoordinateReferenceSystem" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="GeodeticDatum" UniqueId="b4d581f7-612a-4e98-963a-95ad06714397" DefinedType="IfcIdentifier" AttributeFlags="1" />
```

Comment:
GeodeticDatum also in IfcCoordinateReferenceSystem. Is this by design?

Reply:
Good question. @pjanck why do we have two GeodertiDatums?

Reply:
Right, I see. There is a change missing: `GeodeticDatum` and `VerticalDatum` shall be demoted from `IfcCoordinateReferenceSystem` to the sub-entity `IfcProjectedCRS`. Rationale: they are projected CRS specifics and not necessary a part of *any* CRS (see ISO 19111).